### PR TITLE
Add Python one liner to find greatest digit in a number.

### PR DESCRIPTION
**Pull request by u-wut-m8**
**Fixes issue:**

Add Python one liner to find greatest digit in a number.

**Changes:**

### CODE REVIEW

**Comment by u-wut-m8** [comment]
Ok. Thank you for our response.
I will keep that in mind when making further pull requests.

**Comment by Vaibhavi24** [comment]
We could include this as a short-solution of the original program @arnavb

**Comment by arnavb** [comment]
@u-wut-m8 You could create a function in the file called `greatest_digits_with_builtins` that contains your code (and uses a parameter instead).

File: code/mathematical_algorithms/src/greatest_digit_in_number/greatest_digit_in_number.py [hunk 1]
```diff
@@ -13,5 +13,10 @@ def greatest_digit(n):
     return max_digit
 
 
+def greatest_digits_with_builtins(num):
+    print("Greatest digit is: {}".format(int(max([*str(num)]))))
```
Comment (arnavb) [inline code/mathematical_algorithms/src/greatest_digit_in_number/greatest_digit_in_number.py]: Return the value instead.

**Comment by arnavb** [comment]
djldk 2

**Pull request merged**